.github/workflows/nightly-ci.yml

```diff
@@ -111,6 +111,7 @@ jobs:
 
           # 2. Run tests for metrics
           ginkgo e2e/metrics
+          SHARD=shard1 ginkgo e2e/metrics
```

Comment:
I find it interesting that we explicitly run tests twice here and in `e2e-ci.yml`, separating between sharded and unsharded contexts, which made me wonder: would we want to eventually expose the relevant shard (ID or empty value) in labels for metrics? How much sense could it make to users?

Reply:
Ah nice, this is helpful already, thanks!